modules/dcache/src/main/java/org/dcache/services/httpd/HttpdRequestLog.java

```diff
@@ -27,11 +28,23 @@
 class HttpdRequestLog extends AbstractLifeCycle
     implements RequestLog
 {
-    private static final Logger LOGGER
-            = LoggerFactory.getLogger(HttpdRequestLog.class);
+    //private static final Logger LOGGER = LoggerFactory.getLogger(HttpdRequestLog.class);
```

Comment:
You do not need to leave a comment out of the changes. 

git diff is sufficient in providing information in which lines changed..